src/grammar.pegjs

```diff
@@ -3,12 +3,12 @@
   const {
     Assignment,
     BinaryOp,
-    BooleanLiteral,
     Catch,
     ClassDeclaration,
     Closure,
     ConstructorDeclaration,
     FeatureCall,
+    FieldDeclaration,
```

Comment:
Currently we were using the same `VariableDeclaration` for fields, right ?
Now we have a new specific type ?

Also I'm facing annoying the fact that a `VariableDeclaration` has an inner `Variable`, which in turn is the same node as for a "reference".
 Do you think that is needed ? or just brought from xtext version ?

Ideally I would prefer to have
- VariableDeclaration(name, righthand -or whatever, but just name-)
- Reference: could be a reference to a VariableDeclaration, a FieldDeclaration, a Param, etc.